src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -190,17 +190,46 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer.
```

Comment:
The

Reply:
done

---
In reply to: [256222543](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256222543) [](ancestors = 256222543)